### DISCUSSION TITLE
The antivirus has quarantined several dll files during the creation of a new Blank Wrapper

### DISCUSSION THREAD

**Discussion by h3afif** [The-Wineskin-Project/WineskinServer#553] [Q&A] [answered]
AVG Antivirus shows this message for 162 files:
"**_We've put sti.dIl in Quarantine because it was infected with malware:
Win32:SuspBehav-B [Heur]._**"
and this as I begin the creation of a new blank wrapper using the latest Wineskin V1.8.4 CX64Bit engine (installed via Brew).

Am I missing something or do I need to add exceptions to my antivirus?

**Comment by Gcenx** [comment, marked as answer]
https://github.com/Gcenx/WineskinServer#my-antivirus-says-its-a-virus
